internal/provider/resource_application_test.go

```diff
@@ -46,6 +46,10 @@ func TestAccResourceApplicationDeletedOutManually(t *testing.T) {
 
 	orgID := os.Getenv("HUMANITEC_ORG")
 	token := os.Getenv("HUMANITEC_TOKEN")
+	APIHost := os.Getenv("HUMANITEC_HOST")
```

Comment:
Nit: variables should be lowercased
```suggestion
	apiHost := os.Getenv("HUMANITEC_HOST")
```

Reply:
You are right, fixed it!